src/document/advanced.rs

```diff
@@ -1,3 +1,7 @@
+//! Advanced document types
+//!
+//! See [DocumentAdvancedOps](trait.DocumentAdvancedOps.html) for advanced document functions and key terms.
```

Comment:
I like these links to the trait at the top of the module. Helps people find the functions more easily.